RPM/libgcrypt-devel/rpmbuild/SPECS/libgcrypt-devel.spec

```diff
@@ -19,48 +21,52 @@ in GNU Privacy Guard.  This package contains files needed to develop
 applications using libgcrypt.
 
 %install
-mkdir %{buildroot}/bin
-mkdir -p %{buildroot}/usr/include
-mkdir -p %{buildroot}/lib64/pkgconfig
-mkdir %{buildroot}/share
-mkdir %{buildroot}/share/aclocal
-mkdir %{buildroot}/share/info
-mkdir -p %{buildroot}/share/man/man1
-cp /bin/dumpsexp %{buildroot}/bin/
-cp /bin/hmac256 %{buildroot}/bin/
-cp /bin/libgcrypt-config %{buildroot}/bin/
-cp /bin/mpicalc %{buildroot}/bin/
-cp /usr/include/gcrypt.h %{buildroot}/usr/include/
-cp /lib64/libgcrypt.la %{buildroot}/lib64/
-ln -s /lib64/libgcrypt.so.20.2.6 %{buildroot}/lib64/libgcrypt.so
-ln -s /lib64/libgcrypt.so.20.2.6 %{buildroot}/lib64/libgcrypt.so.20
-cp /lib64/libgcrypt.so.20.2.6 %{buildroot}/lib64/
-cp /lib64/pkgconfig/libgcrypt.pc %{buildroot}/lib64/pkgconfig/
-cp /share/aclocal/libgcrypt.m4 %{buildroot}/share/aclocal/
-cp /share/info/dir %{buildroot}/share/info/
-cp /share/info/gcrypt.info %{buildroot}/share/info/
-cp /share/info/gcrypt.info-1 %{buildroot}/share/info/
-cp /share/info/gcrypt.info-2 %{buildroot}/share/info/
-cp /share/man/man1/hmac256.1 %{buildroot}/share/man/man1/
+mkdir -p %{buildroot}/opt/awn/bin
+mkdir -p %{buildroot}/opt/awn/usr/include
+mkdir -p %{buildroot}/opt/awn/lib64/pkgconfig
+mkdir -p %{buildroot}/opt/awn/share
+mkdir -p %{buildroot}/opt/awn/share/aclocal
+mkdir -p %{buildroot}/opt/awn/share/info
+mkdir -p %{buildroot}/opt/awn/share/man/man1
+mkdir -p %{buildroot}/etc/ld.so.conf.d
+cp /bin/dumpsexp %{buildroot}/opt/awn/bin/
+cp /bin/hmac256 %{buildroot}/opt/awn/bin/
+cp /bin/libgcrypt-config %{buildroot}/opt/awn/bin/
+cp /bin/mpicalc %{buildroot}/opt/awn/bin/
+cp /usr/include/gcrypt.h %{buildroot}/opt/awn/usr/include/
+cp /lib64/libgcrypt.la %{buildroot}/opt/awn/lib64/
+ln -s /lib64/libgcrypt.so.20.2.6 %{buildroot}/opt/awn/lib64/libgcrypt.so
+ln -s /lib64/libgcrypt.so.20.2.6 %{buildroot}/opt/awn/lib64/libgcrypt.so.20
```

Comment:
For completeness, I am going to leave a comment here. Just want to make sure we know the result of changing the links.

Reply:
This is actually a bug. Well caught. The libgcrypt.so.20.2.6 file should be located in /opt/awn/lib64, not /lib64

Reply:
You are correct. This is actually a bug:
# ls -l /opt/awn/lib64/libgcr*
```-rwxr-xr-x. 1 root root     954 Sep 23 16:56 /opt/awn/lib64/libgcrypt.la
lrwxrwxrwx. 1 root root      26 Sep 23 18:04 /opt/awn/lib64/libgcrypt.so -> /lib64/libgcrypt.so.20.2.6
lrwxrwxrwx. 1 root root      19 Sep 23 18:25 /opt/awn/lib64/libgcrypt.so.20 -> libgcrypt.so.20.2.6
-rwxr-xr-x. 1 root root 4586416 Sep 23 16:56 /opt/awn/lib64/libgcrypt.so.20.2.6```